src/views/Login.jsx

```diff
@@ -0,0 +1,30 @@
+import { SignInButton, SignOutButton } from '../api/useAuth';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function Login({ user }) {
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		if (user) {
+			navigate('/');
+		}
+	}, [user]);
+
+	return (
+		<div className="flex justify-center items-center h-screen">
```

Comment:
I would
- add a shadow to make it look like the sign in container pops out a bit and add `rounded-xl` as `rounded-lg` doesn't seem to work in our project.
- Reduce the hight and width of the container with  `w-11/12 max-w-lg p-8`(instead of `h-2/4 w-5/12`)
and then add a margin bottom on the img container of `mb-6`(instead of `h-2/4
`). 
- include the logo image(s) provide instead of the one in the file now.
- Increase the padding `p-2` on the  button and add a hover with opacity `bg-pink-500 hover:bg-pink-500 hover:bg-opacity-80`
- 
![Screenshot 2024-10-05 at 11 45 16](https://github.com/user-attachments/assets/7c2d0017-1bd2-4ab3-b32d-e43c6b2b879d)